django-stubs/db/models/base.pyi

```diff
@@ -71,22 +71,22 @@ class Model(metaclass=ModelBase):
     def get_constraints(self) -> list[tuple[type[Model], Sequence[BaseConstraint]]]: ...
     def save(
         self,
-        force_insert: bool = ...,
+        force_insert: bool | tuple[ModelBase] = ...,
```

Comment:
By looking at the `Model._validate_force_insert` classmethod here: https://github.com/django/django/blob/476d7c581ac97dae2bc14ee79a1294db4f802e74/django/db/models/base.py#L843-L861 I think it's a tuple containing an arbitrary amount of model instances.

Suspect it was just a typo

```suggestion
        force_insert: bool | tuple[ModelBase, ...] = ...,
```

Reply:
Oh, that's right. I forgot to add ellipsis 😂  Thanks, I fixed at 847c2ae065a5239c8393849309d6b0e8e5dcf08b.